source/views/settings/sections/odds-and-ends.js

```diff
@@ -28,11 +46,21 @@ class OddsAndEndsSection extends React.PureComponent<Props> {
       url: 'https://github.com/StoDevX/AAO-React-Native',
       id: 'ContributingView',
     })
+  onAppIconButton = () => this.onPressButton('IconSettingsView')
 
   render() {
     return (
       <View>
         <Section header="MISCELLANY">
+          {this.state.supported ? (
+            <Cell
```

Comment:
Why use `<Cell/>` here instead of `<PushButtonCell/>`?

Reply:
Ah, because you have `cellStyle="RightDetail"`.

Reply:
But you don't have a `detail=` prop, so I'd like to request that this change to `PushButtonCell`.